File: 프로그래머스/0/120902. 문자열 계산하기/문자열 계산하기.swift
import Foundation

func solution(_ my_string:String) -> Int {
    var arr = my_string.split(separator: " ")
    var answer: Int = Int(arr[0])!
    
    for i in 0..<arr.count {
        if arr[i] == "+" {
            answer += Int(arr[i+1])!
        }
        
        if arr[i] == "-" {
            answer -= Int(arr[i+1])!
        }
    }
    
    return answer
}

File: 프로그래머스/0/120902. 문자열 계산하기/문자열 계산하기.py
def solution(my_string):
    # 홀수 -> 숫자, 짝수 -> 기호
    arr = my_string.split()
    answer = int(arr[0])

    
    for i in range(len(arr)):
        if arr[i] == "+":
            answer += int(arr[i+1])
        elif arr[i] == "-":
            answer -= int(arr[i+1])
        print(answer)
    
    return answer